src/Templates/src/templates/maui-multihead/MauiApp.1/MauiApp.1.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>DOTNET_TFM</TargetFramework>
```

Comment:
I know this is a multi-project template... But can't the shared lib be multi-target? That will allow users to use the `PlatformBehavior` nicely, and also access platform-specific APIs. I believe having this a multi-target will not break the multi-project structure

Reply:
@davidortinau did you have any preference?